docs/Handling_Site_Failure.md

```diff
@@ -5,14 +5,13 @@ In a geographically redundant deployment, you may encounter the situation where
 * How to recover from this situation
 * The likely impact of a site failure
 
-More information about Clearwater's geographic redundancy support is available [here](http://clearwater.readthedocs.io/en/latest/docs/Geographic_redundancy.html) and informaion on how to configure a geographically redundant deployment is available [here](http://clearwater.readthedocs.io/en/latest/Configuring_GR_deployments.html).
+More information about Clearwater's geographic redundancy support is available [here](http://clearwater.readthedocs.io/en/latest/Geographic_redundancy.html) and information on how to configure a geographically redundant deployment is available [here](http://clearwater.readthedocs.io/en/latest/Configuring_GR_deployments.html).
 
 ### Recovery
 
 To recover from this situation, all you need to do is remove the failed Vellum nodes from Cassandra cluster.
 
-    * Determine the site ID of the failed site (this would be ). You can also find this by running `cw-run-in-signaling-namespace nodetool status`, and 
```

Comment:
It wasn't meant to be there in the first place. I would also expect users to know the site ID